integration/logging_test.go

```diff
@@ -0,0 +1,147 @@
+package integration_test
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/sclevine/spec"
+
+	. "github.com/onsi/gomega"
+	"github.com/paketo-buildpacks/occam"
+	. "github.com/paketo-buildpacks/occam/matchers"
+)
+
+func testLogging(t *testing.T, context spec.G, it spec.S) {
+
+	var (
+		Expect = NewWithT(t).Expect
+
+		pack   occam.Pack
+		docker occam.Docker
+
+		image       occam.Image
+		secondImage occam.Image
+		name        string
+		source      string
+	)
+
+	it.Before(func() {
+		pack = occam.NewPack()
+		docker = occam.NewDocker()
+		var err error
+		name, err = occam.RandomName()
+		Expect(err).NotTo(HaveOccurred())
+	})
+
+	it.After(func() {
+		Expect(docker.Image.Remove.Execute(image.ID)).To(Succeed())
+		Expect(docker.Image.Remove.Execute(secondImage.ID)).To(Succeed())
```

Comment:
This expectation fails is secondImage is not set in every it block.

Reply:
Resolved